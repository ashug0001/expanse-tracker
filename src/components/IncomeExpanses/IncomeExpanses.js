import React from "react";
import PropTypes from "prop-types";

const IncomeExpanses = () => {
  return (
    <div class="inc-exp-container">
      <div>
        <h4>Income</h4>
        <p id="money-plus" class="money plus">
          +$0.00
        </p>
      </div>
      <div>
        <h4>Expense</h4>
        <p id="money-minus" class="money minus">
          -$0.00
        </p>
      </div>
    </div>
  );
};

IncomeExpanses.propTypes = {};

export default IncomeExpanses;
