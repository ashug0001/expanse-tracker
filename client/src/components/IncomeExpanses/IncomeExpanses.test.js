import React from "react";
import { shallow } from "enzyme";

import IncomeExpanses from "./IncomeExpanses";

describe("IncomeExpanses", () => {
  it("should render IncomeExpanses", () => {
    const wrapper = shallow(<IncomeExpanses />);
  });

  it("should render initial layout", () => {
    const component = shallow(<IncomeExpanses title="Expanse Tracker" />);

    expect(component.getElement()).toMatchSnapshot();
  });
});
