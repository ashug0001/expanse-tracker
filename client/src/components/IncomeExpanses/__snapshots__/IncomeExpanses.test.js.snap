// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IncomeExpanses should render initial layout 1`] = `
<div
  class="inc-exp-container"
>
  <div>
    <h4>
      Income
    </h4>
    <p
      class="money plus"
      id="money-plus"
    >
      +$0.00
    </p>
  </div>
  <div>
    <h4>
      Expense
    </h4>
    <p
      class="money minus"
      id="money-minus"
    >
      -$0.00
    </p>
  </div>
</div>
`;
